notebooks/gammanu.ipynb

```diff
@@ -0,0 +1,1105 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    gammanu = simplify(F_nu.doit())
</pre><p>Maybe call this <code>F_nu_expanded</code>? The name <code>gammanu</code> sounds like another of the gamma functions   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/graphcore-research/pyscf-ipu/pull/119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>